dclab/definitions/meta_const.py

```diff
@@ -124,6 +125,25 @@
         ["target duration", float, "Target measurement duration [min]"],
         ["target event count", fint, "Target event count for online gating"],
     ],
+    # All qpi-related keywords
+    "qpi": [
+        # experiment-related qpi metadata
+        ["wavelength", float, "Laser wavelength [nm]"],
+        ["medium index", float, "Refractive index of Medium"],
+        # post-analysis-related qpi metadata
+        ["software version", str, "Software version used"],
+        # calculation of pha and amp from hologram
+        ["filter_name", str, "Fourier filter used"],
+        ["filter_size", str, "Fourier filter size [1/pix]"],
+        ["filter_size_interpretation", str, "How to interpret filter size"],
+        ["sideband_freq", f2dfloatarray,
```

Comment:
I assumed this was the correct type for a tuple of floats.

Reply:
:thinking:  not really. I would add a separate conversion function that checks that this is actually **two** floats.